### DISCUSSION TITLE
[Feature request] Adding query options Format and Type for Google Cloud Monitoring datasource

### DISCUSSION THREAD

**Discussion by GedPL** [grafana/grafana#57441] [Data Sources]
Hi Team,
What I want to achieve is showing multiple metrics (last value) for each server in separate columns in Table Panel as shown below. 

![image](https://user-images.githubusercontent.com/31762538/197201764-4e2ee471-fbb7-4728-bae2-9658a748d380.png)

With Prometheus data source it's possible using query options:

![image](https://user-images.githubusercontent.com/31762538/197201030-591c52c9-f75b-49af-be0c-7a41611fb578.png)

and Merge transformation.

Could you please consider adding such query options for Google Cloud Monitoring datasource ?

I've tried to do that using various transformations combinations only but with no success...

Thanks a lot !
Ged

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
